containers/ddev-webserver/ddev-webserver-base-files/etc/bashrc/commandline-addons.bashrc

```diff
@@ -1,4 +1,6 @@
-export PATH="~/bin:$PATH:/var/www/html/vendor/bin:/var/www/html/bin:$DDEV_COMPOSER_ROOT/vendor/bin"
+PATH="$PATH:/var/www/html/vendor/bin:/var/www/html/bin:$DDEV_COMPOSER_ROOT/vendor/bin"
```

Comment:
Perhaps slightly better

```suggestion
PATH="${PATH}:/var/www/html/vendor/bin:/var/www/html/bin:${DDEV_COMPOSER_ROOT}/vendor/bin"
```

However, I don't understand the removal of ~/bin. I think people might add things there with homeadditions.

Reply:
> I don't understand the removal of `~/bin`

I didn't remove it, see the line below. I preserved the original order, so the binaries inside `N_PREFIX` will not override something in `~/bin`.